src/Album.jsx

```diff
@@ -0,0 +1,22 @@
+import { ArtistName } from "./ArtistName";
+import { AlbumName } from "./AlbumName";
+import { CoverImage } from "./CoverImage";
+//Component (function in react). Working with bigger amounts of code, you need to work in diffrent files (components).
+export const Album = (props) => {
+  return (
+    <div>
```

Comment:
Consider using a React fragment instead of a div, if it's just for wrapping purposes